plist

```diff
@@ -761,7 +761,7 @@
 /usr/local/opnsense/scripts/filter/list_states.py
 /usr/local/opnsense/scripts/filter/list_table.py
 /usr/local/opnsense/scripts/filter/list_tables.py
-/usr/local/opnsense/scripts/filter/pfstatistcs.py
+/usr/local/opnsense/scripts/filter/pfstatistics.py
```

Comment:
I'm going to stop review here. We can't just run a blind spell correction over the code base. The only useful way here is to either fix these things manually as it requires renaming a file and changing the path accordingly where used or leaving out operational changes and just submitting comments and text adjustments first.

Touching function names is similarly dangerous as they also need to be checked in plugins.git at the very least and should also be submitted individually.

Reply:
I do this kind of PR regularly and am happy to accommodate whichever model is preferred.

For this specific item, the corresponding rename is here:
https://github.com/opnsense/core/pull/5885/files#diff-cfb912843b917b2ad2e9cdece92d8ded5c94e933a18f735b7b0b1b6fe11e239f

Reply:
For the time being, I've dropped the things which sounded like they'd be most concerning.

They roughly correspond to items I've flagged below (identifiable by `Outdated`)

Reply:
I should note that the reason there are so many individual commits is so that one can easily review scary ones, or so I can easily drop scary/objectionable ones. (I hate squashing early, because it makes pealing things apart risky.)

Reply:
I've force pushed a set which I think is safe.

I'm happy to let you drive the PR from here.

I've left the original commits here:
https://github.com/jsoref/opnsense-core/tree/spelling-full

And yeah, as long as someone reminds me, I can rebase the remainder of that (minus the things that really belong to upstreams) and make PR(s) for them.